desc/compute/utils.py

```diff
@@ -1365,3 +1365,38 @@ def body(i, mins):
     # The above implementation was benchmarked to be more efficient than
     # alternatives without explicit loops in GitHub pull request #501.
     return grid.expand(mins, surface_label)
+
+
+# defines the order in which objective arguments get concatenated into the state vector
+arg_order = (
```

Comment:
This was deleted a while ago but seems to keep coming back. Nothing references `arg_order` anymore so it should be removed

Reply:
Ok, that part always causes merge conflicts, probably that's why I didn't realize it.